typeDefs.js

```diff
@@ -407,11 +405,12 @@ const typeDefs = gql`
     deleteUser(id: ID!): User
     updateUserById(user: UserInput!, id: ID!): User
     updateUserByToken(user: UserInput!): User
-    confirmUser(token: String!): Boolean
+    confirmUser(token: String!): User
```

Comment:
Please, rename it to the `confirmUserEmail` and don't return the user back because it can be done by not logged in user

Reply:
I need to update user "confirmed: field"

Reply:
fixed